data/tribes/initialization/frisians/units.lua

```diff
@@ -1385,7 +1385,7 @@ wl.Descriptions():new_tribe {
       {
          name = "wheatfield_medium",
          helptexts = {
-            -- TRANSLATORS: Helptext for an empire/barbarian immovable usable by frisians: Wheat field
+            -- TRANSLATORS: Helptext for an Empire/barbarian immovable usable by frisians: Wheat field
```

Comment:
```suggestion
            -- TRANSLATORS: Helptext for an Empire/Barbarian immovable usable by Frisians: Wheat field
```

Reply:
This PR is for the Empire, the one for the Frisians will go next... 😄 , and the change for Barbarian is already included [here](https://github.com/widelands/widelands/pull/5910/files#diff-2a1dac2825a3dda7258dc30da855da9073e5db4858d48bf825ccf3facf3eecd1)

Reply:
I understand the intention, but for drive by fixes in the same line or block it might be better to do them on the fly, because otherwise you get conflicts as it is the case now.
we are not that pedantic on such drive by fixes